src/ncp/ncp_base.cpp

```diff
@@ -2163,7 +2163,7 @@ otError otNcpStreamWrite(int aStreamId, const uint8_t *aDataPtr, int aDataLen)
 
 extern "C" void otNcpPlatLogv(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat, va_list aArgs)
 {
-    char logString[128];
+    char logString[OPENTHREAD_CONFIG_NCP_SPINEL_LOG_MAX_SIZE];
```

Comment:
Should the code below add a '\0' to logstring[sizeof(logsstring) - 1] if it truncates the length?

Reply:
`vsnprintf` would add a `\0` (http://www.cplusplus.com/reference/cstdio/vsnprintf/).